frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -91,7 +91,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 )
               }
               {
-                hasRole(currentUser, "ROLE_ADMIN")  && (
+                hasRole(currentUser, "ROLE_ADMIN") && !hasRole(currentUser, "ROLE_DRIVER") && (
```

Comment:
Hi professor, I believe this is because this tab is mainly for drivers, not admins. The endpoint used in this driver availability is not the admin one. Admin can use the other tab in Admin drop-down, that has the highest privilege.

The only reason this tab still is visible to admins is because Shifts does not have an admin special page yet.

![Screenshot_20240606-163611](https://github.com/ucsb-cs156-s24/proj-gauchoride-s24-5pm-5/assets/36137654/fe25e3e6-87ca-4e3a-953f-04141c9b35f6)